edx/edx-app-ios
# Handle Dynamic Type Text Change for Course Discovery

### Description

[LEARNER-6733](https://openedx.atlassian.net/browse/LEARNER-6733)

This PR handles dynamic text change of iOS App Settings for the authenticated web content. `AuthenticatedWebViewController` is now listing to `UIContentSizeCategoryDidChangeNotification` and `reload` itself whenever value of dynamic text changes in iOS App Settings.

### Notes

### How to test this PR
- Access Course Discovery for logged in or logged out user
- Open iOS App Settings
- Navigate to General -> Accessibility -> Large Text
- Change the value of the slider
- Reopen/comeback to the edX app
- Course Discovery should reload itself and shows the content according to the newly selected dynamic text size value.
